src/timing.h

```diff
@@ -29,17 +29,37 @@ void jl_timing_block_stop(jl_timing_block_t *cur_block);
 #define HAVE_TIMING_SUPPORT
 #elif defined(_COMPILER_GCC_)
 #define HAVE_TIMING_SUPPORT
+#elif defined(_COMPILER_MINGW_)
```

Comment:
@vtjnash I just noticed you got rid of this macro in https://github.com/JuliaLang/julia/pull/34471, I guess this can go?

Reply:
Yes